frontend/src/pages/Clients.vue

```diff
@@ -22,48 +30,64 @@
         <q-btn @click="row_expand(props.row)" flat icon="aspect_ratio" />
       </q-td>
     </q-table>
-    
+    <!-- Email Client Button -->
+    <div class="q-pa-md email-btn">
+      <q-btn
+        icon="email"
+        label="Email Selected Clients"
+        stack
+        color="accent"
+        style="padding: 7px;"
+        @click="getSelectedEmail"
+      />
+
+      <q-dialog v-model="showEmailPopup">
+        <EmailPopup
+          :selected="selected"
+          :allEmails="allEmails"
+          @dialog-closed="showEmailPopup = false"
+        />
+      </q-dialog>
+    </div>
   </div>
 </template>
 
 <script>
-import axios from "axios";
 import ClientPopup from "../components/ClientPopup.vue";
+import EmailPopup from "../components/EmailPopup.vue";
+
 export default {
```

Comment:
nit: add a Name to the top of all component exports, e.g. `name: "Clients"` for this component